daemons/gptp/common/ptp_message.cpp

```diff
@@ -933,7 +937,7 @@ void PTPMessageFollowUp::processMessage(IEEE1588Port * port)
 	sync_arrival = sync->getTimestamp();
 
 	delay = port->getLinkDelay();
-	if ((delay = port->getLinkDelay()) == 3600000000000) {
+	if ((delay = port->getLinkDelay()) == 3600000000000) { /*TODO: ONE_WAY_DELAY_DEFAULT*/
```

Comment:
So I guess this would do:

-       delay = port->getLinkDelay();
-       if ((delay = port->getLinkDelay()) == 3600000000000) { /*TODO: ONE_WAY_DELAY_DEFAULT*/
+       if( !port->getLinkDelay(&delay) ) {
                goto done;